modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js

```diff
@@ -64,6 +65,28 @@ continueQuoteButton.onclick = function () {
 		});
 };
 
+getQuoteForm.onsubmit = function (e) {
```

Comment:
Please, change the parameter name: e -> event

Reply:
Alright